README.md

```diff
@@ -1,7 +1,11 @@
 # ChainerUI
 
+![training_captures](https://user-images.githubusercontent.com/414255/34244148-637e6630-e667-11e7-8a6b-9a857fa806bc.gif)
+
 ChainerUI is a visualization and management tool for [Chainer](https://github.com/chainer/chainer).
 
+[![CircleCI](https://circleci.com/gh/chainer/chainerui.svg?style=svg)](https://circleci.com/gh/chainer/chainerui)
```

Comment:
I like ![](https://circleci.com/gh/chainer/chainerui.svg?style=shield) more than ![](https://circleci.com/gh/chainer/chainerui.svg?style=svg).

Reply:
Would you like to swap the position of CI badge and capture image?

ex.
```
# Header
![Badge]()
Short description
![Capture]
...
```

Reply:
OK, I'll fix